src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -141,6 +141,8 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
   const isGuestEnabled = isGuestRoom || isGuestAndServicesRoom;
   const isServicesEnabled = isServicesRoom || isGuestAndServicesRoom;
 
+  const isCellsFeatureEnabled = Config.getConfig().FEATURE.ENABLE_CELLS;
```

Comment:
Just a small suggestion:

```suggestion
  const isCellsEnabled = Config.getConfig().FEATURE.ENABLE_CELLS && isCellsOptionEnabled;
```